.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -150,16 +150,16 @@ AddType text/vtt                            vtt
 
     # Redirects for "latest" version
     RewriteRule ^(container-platform|enterprise)/?$ /container-platform/latest [R=301]
-    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.9/$2 [NE,R=301]
+    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.10/$2 [NE,R=301]
     RewriteRule ^(online)/(3\.0|3\.1|3\.2|3\.3|3\.4|3\.5|3\.6|3\.7|3\.9|3\.10|3\.11|latest)/?(.*)$ /$1/pro/$3 [NE,R=301]
 
     # Release notes redirects
     RewriteRule ^container-platform\/4\.(.+)\/release_notes\/ocp-4-((?!\1).+)-release-notes.html /container-platform/4.$1/release_notes/ocp-4-$1-release-notes.html [NE,R=301]
     RewriteRule ^container-platform\/3\.(.+)\/release_notes\/ocp_3_((?!\1).+)_release_notes.html /container-platform/3.$1/release_notes/ocp_3_$1_release_notes.html [NE,R=301]
 
     # Redirects for the Updating book
-    RewriteRule ^container-platform/(4\.6|4\.7|4\.8|4\.9)/updating/updating-cluster-between-minor.html /container-platform/$1/updating/updating-cluster-within-minor.html [NE,R=302]
-    RewriteRule ^container-platform/(4\.6|4\.7|4\.8|4\.9)/updating/updating-cluster.html /container-platform/$1/updating/updating-cluster-within-minor.html [NE,R=302]
+    RewriteRule ^container-platform/(4\.6|4\.7|4\.8|4\.9|4\.10)/updating/updating-cluster-between-minor.html /container-platform/$1/updating/updating-cluster-within-minor.html [NE,R=302]
```

Comment:
@bobfuru hi! For this rule and the one in the line below it, it looks like the original pages don't exist on 4.10. Both of these give a 404:

- https://docs.openshift.com/container-platform/4.10/updating/updating-cluster-between-minor.html
- https://docs.openshift.com/container-platform/4.10/updating/updating-cluster.html

Since we have never released content on these URLs in a GA docs version, what is the reason for adding the redirect for 4.10? Have we published these 4.10 URLs somewhere publicly?